### PR TITLE
UPDATED

### PR DESCRIPTION
**Pull request by alax99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# :rainbow: The React:laughing: Project: [![Coverage Status](https://coveralls.io/repos/github/zixuan75/react-project/badge.svg)](https://coveralls.io/github/zixuan75/react-project?branch=master)
+# :rainbow: The React:laughing: Project:star:stars:: [![Coverage Status](https://coveralls.io/repos/github/zixuan75/react-project/badge.svg)](https://coveralls.io/github/zixuan75/react-project?branch=master)
```
Comment (zixuan75) [inline README.md]: I think this is too many emojis.

Comment (alax99) [reply README.md]: I agree.

Comment (zixuan75) [reply README.md]: Made one.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# :rainbow: The React:laughing: Project: [![Coverage Status](https://coveralls.io/repos/github/zixuan75/react-project/badge.svg)](https://coveralls.io/github/zixuan75/react-project?branch=master)
+# :rainbow: The React:laughing: Project:stars:: [![Coverage Status](https://coveralls.io/repos/github/zixuan75/react-project/badge.svg)](https://coveralls.io/github/zixuan75/react-project?branch=master)
```
Comment (zixuan75) [inline README.md]: That's ok for me.

**Pull request merged**